conda-build/linux_matrix_builder.py

```diff
@@ -38,6 +44,7 @@ def main():
         py_vers, bullet_modes, headless_modes, cuda_vers
     ):
         env = os.environ.copy()
+        env["VERSION"] = habitat_sim.__version__ + time.strftime(".%Y.%m.%d") # including a timestamp in anticipation of nightly builds
```

Comment:
Why not just do a has of the current commit? No reason to add builds that are identical.

Reply:
Requires gitpython library.
```python
import git
repo = git.Repo(search_parent_directories=True)
sha = repo.head.object.hexsha
```